quinn-proto/src/lib.rs

```diff
@@ -59,6 +59,9 @@ pub use crate::transport_error::{Code as TransportErrorCode, Error as TransportE
 
 pub mod congestion;
 
+mod cid_generator;
+pub use crate::cid_generator::ConnectionIdGenerator;
```

Comment:
We should reexport `RandomConnectionIdGenerator` to allow easy customization of connection ID length.